e2e/pnpm_lockfiles/base/package.json

```diff
@@ -3,19 +3,27 @@
     "private": true,
     "dependencies": {
         "@aspect-test/a": "^5.0.2",
+        "// @aspect-test/a2": "npm: reference to another dependency, not specifying the version number",
```

Comment:
you can have comment deps?

Reply:
Ahhh, is it only outside `dependencies`? I think you're right 😢

Reply:
I was hoping to comment **somewhere** to say why we have `"is-odd": "npm:is-odd"`, and explaining the other test cases is also valuable. Mainly to ensure no one accidentally removes the test cases in the future...